### ISSUE TITLE
Breaks Keyboard Search extension

### ISSUE THREAD

**Issue by neilio** [marcocebrian/kagisearchsafari#3]
This extension conflicts with the Keyboard Search extension. When both are installed and active Kagi intercepts keyword searches so they don't work. https://apps.apple.com/ca/app/keyword-search/id1558453954

**Comment by jimeh** [comment]
I've got the same issue, and for the time being I consider Keyword Search more important than Kagi, for better or worse.

**Comment by jimeh** [comment]
I should mention, that the iOS/iPadOS variant of the Kagi Extension, does not have this issue. On iOS I can use Kagi in Safari, and still use Keyword Search too.

iOS/iPadOS extension: https://apps.apple.com/app/kagi-search-for-safari/id1607766153

**Comment by jimeh** [comment]
Actually, after further testing, Keyword Search doesn't work anymore on iOS either when using the Kagi extension.

For now, I've worked around it in a pretty hacky way with a userscript that essentially detects searches on DuckDuckGo and redirects to Kagi. I picked DuckDuckGo as it's probably the lightest one to load.

For anyone who's curious, my userscript is here: https://github.com/jimeh/dotfiles/blob/main/userscripts/kagi-for-safari.user.js

And I'm using the Userscripts ([App Store](https://itunes.apple.com/us/app/userscripts/id1463298887)/[GitHub](https://github.com/quoid/userscripts)) app macOS, iOS, and iPadOS to make it work.

With that in place, simply setting the search engine to DuckDuckGo in Safari will then redirect searches to Kagi, most of the time fast enough that you don't really notice it.
